Makefile

```diff
@@ -1,35 +1,46 @@
 .PHONY: setup release clean clean-all
 
 setup:
-	cargo install cross
-	npm install -g yarn
+	# Start setup
+	@if ! command -v cross &> /dev/null; then \
+			cargo install cross --git https://github.com/cross-rs/cross; \
+	fi
+	yarn
```

Comment:
could be improved. first check that yarn is installed and if not use npm -i instead of yarn